src/website/app/demos/Tooltip/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import { componentTheme } from '../../../../Tooltip/Tooltip';
+import bestPractices from './bestPractices';
+import examples from './examples';
+
+const doc = require('!!react-docgen-loader!../../../../Tooltip/Tooltip');
+
+export default {
+  bestPractices,
+  componentTheme,
+  doc,
+  examples,
+  slug: 'tooltip',
+  title: 'Tooltip',
+  whenHowToUse: `Tooltips are Popovers that display information related to an element when the element receives keyboard focus or the mouse hovers over it. It appears after a small dealy and disappears when \`Escape\` is pressed or on mouse out.
+
+Tooltips do not receive focus. A hover that contains focusable elements can be made using a non-modal dialog.`
```

Comment:
I don't understand that last sentence.

Reply:
This was pulled from the [aria docs](https://www.w3.org/TR/wai-aria-practices/#tooltip). I'm not sure how to convey what people should do if they want focusable elements to be presented on hover.

Reply:
Ahh, I see. I would argue focusable elements within an on-hover-only container is bad practice. Let's just remove it altogether.